Digipost/APIClient.swift

```diff
@@ -445,6 +452,8 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
                 }, failure: { (error) -> Void in
                     if error.code == Int(SHCOAuthErrorCode.InvalidRefreshTokenResponse.rawValue) {
                         self.deleteRefreshTokensAndLogoutUser()
+                    } else {
+                        Logger.dpostLogError("Deleting oauth tokens and logout user. Not invalid refresh token.  error: \(error)")
```

Comment:
I did a small rewrite. I either want to logout user if refreshtoken is out, or i want to to let the failure block handle the error. If the error is shown to user, it's logged by Logger there instead. 
commit: 0edc84112b8ada730a3ffb62f5aef600b56e9a65